src/angular-versions.ts

```diff
@@ -59,6 +59,64 @@ export const versions = new Map<string, PackageJsonVersion>([
       }
     }
   ],
+  [
+    '8.2.x',
+    {
+      dependencies: {
+        '@angular/animations': '~8.2.0',
+        '@angular/common': '~8.2.0',
+        '@angular/compiler': '~8.2.0',
+        '@angular/core': '~8.2.0',
+        '@angular/forms': '~8.2.0',
+        '@angular/platform-browser': '~8.2.0',
+        '@angular/platform-browser-dynamic': '~8.2.0',
+        '@angular/router': '~8.2.0',
+        rxjs: '~6.4.0',
+        tslib: '^1.10.0',
+        'zone.js': '~0.9.1'
+      },
+      devDependencies: {
+        '@angular-devkit/build-angular': '~0.802.0',
+        '@angular-devkit/build-ng-packagr': '~0.802.0',
+        '@angular/cli': '~8.2.0',
```

Comment:
Let's use the latest compatible CLI and builders
```suggestion
        '@angular-devkit/build-angular': '~0.803.0',
        '@angular-devkit/build-ng-packagr': '~0.803.0',
        '@angular/cli': '~8.3.0',
```